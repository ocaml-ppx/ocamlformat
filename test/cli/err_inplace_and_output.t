  $ echo 'let x = 1' > a.ml

  $ ocamlformat --inplace --output o.ml a.ml
  ocamlformat: Cannot specify --output with --inplace
  [1]
