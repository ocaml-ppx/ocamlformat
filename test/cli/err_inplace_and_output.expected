ocamlformat: Cannot specify --output with --inplace
